packages/webdriverio/src/utils.js

```diff
@@ -82,7 +82,7 @@ export function getBrowserObject (elem) {
 /**
  * transform whatever value is into an array of char strings
  */
-export function transformToCharString (value) {
+export function transformToCharString (value, { translateToUnicode } = { }) {
```

Comment:
given this is an internal utility I would advocate for this structure:
```suggestion
export function transformToCharString (value, translateToUnicode) {
```

Reply:
I agree with you. Should we define the default value for the new property as I did ?

Reply:
default property should mimic current behavior so we don't break user tests.

Reply:
Thanks, I'll update it